test/api_projects_test.go

```diff
@@ -79,6 +81,8 @@ func Test_client_ProjectsAPIService(t *testing.T) {
 
 			resp, httpRes, err := apiClient.ProjectsAPI.GetProjectAccess(context.Background(), projectId).Execute()
 
+			fmt.Println(err)
```

Comment:
Not salient but gosh darn does it making debugging this test easier. I'd like to keep it